provider/src/main/java/com/raygun/raygun4android/rum/RUM.java

```diff
@@ -144,8 +144,7 @@ private void sendRUMEvent(String eventName, RaygunUserInfo userInfo) {
                 timestamp = df.format(c.getTime());
             }
 
-            RaygunUserInfo user =
-                    userInfo == null ? new RaygunUserInfo(null, null, null, null) : userInfo;
+            RaygunUserInfo user = userInfo == null ? RaygunUserInfo.anonymousSync() : userInfo;
```

Comment:
Using `anonymousSync()` is a temporary solution, I will switch to the suspend function when `RUM.java` gets converted to Kotlin